src/main/groovy/org/mockito/release/internal/DefaultEnvPropertyAccessor.java

```diff
@@ -0,0 +1,11 @@
+package org.mockito.release.internal;
+
+import org.mockito.release.internal.util.EnvPropertyAccessor;
+
+public class DefaultEnvPropertyAccessor implements EnvPropertyAccessor{
```

Comment:
Nitpicky:
- I would probably not create an interface because it's an internal env variable wrapper so we can keep things simple
- Use "env variable" instead of "env property" in the name. Example class name: "EnvVariables".
- Add tiny javadoc code comment on the class explaining why we're wrapping access to env variables